kctf-operator/operator.yaml

```diff
@@ -0,0 +1,4549 @@
+apiVersion: v1
```

Comment:
there's another operator.yaml above, what's the difference?

Reply:
this one could be `.gitignore`

Reply:
added to .gitignore